awesome.md

```diff
@@ -0,0 +1,31 @@
+# Awesome CRI-O [![Awesome](https://awesome.re/badge.svg)](https://awesome.re)
+
+This list follows the [awesome
+manifesto](https://github.com/sindresorhus/awesome/blob/master/awesome.md), and
+collects every useful resource about CRI-O.
+
+## Contents
+
+- [Resources](#resources)
+  - [Articles](#articles)
+  - [Community](#community)
+  - [Videos](#videos)
+  - [Websites](#websites)
+
+## Resources
+
+### Articles
+
+- [CentOS 7 with CRI-O on EKS (Nov 20, 2018)](https://medium.com/errnothxbye/centos-7-with-cri-o-on-eks-ae9684aff764)
+
+### Community
+
+- [Kubernetes Slack #cri-o](https://slack.com/app_redirect?channel=CAZH62UR1)
+
+### Videos
+
+- [CRI-O: All the Runtime Kubernetes Needs](https://www.youtube.com/watch?v=NVa8xR4Eyj8)
+
+### Websites
+
+- [cri-o.io (official website)](https://cri-o.io/)
```

Comment:
I'm not sure if this list belongs here or on the cri-o.io website, but if it does end up here, we ought to add a link from the site to this page.  The advantage of keeping it here is it would more likely be updated by contributors.